app/bundles/CoreBundle/EventListener/RouterSubscriber.php

```diff
@@ -29,15 +26,9 @@ class RouterSubscriber implements EventSubscriberInterface
      */
     private $host;
 
-    /**
-     * @var string|null
-     */
-    private $httpsPort;
+    private string|int $httpsPort;
```

Comment:
How can Rector know that it can be either string or int but not anything else? The constructor has no type defined for this.

Reply:
I'll check, it's also possible I added it manually...

...so I based this typed based on parameter from ENV:

* https://github.com/mautic/mautic/blob/dcba9688668eb73fe9f840848dadd0bd67a22d58/app/config/parameters.php#L55-L56

That is being passed in constructor of this service:

* https://github.com/mautic/mautic/blob/dcba9688668eb73fe9f840848dadd0bd67a22d58/app/bundles/CoreBundle/Config/config.php#L177-L178